src/i18n/en/nav.ts

```diff
@@ -46,56 +46,52 @@ export default [
 	},
 	{ text: 'Pages', slug: 'core-concepts/astro-pages', key: 'core-concepts/astro-pages' },
 	{ text: 'Layouts', slug: 'core-concepts/layouts', key: 'core-concepts/layouts' },
-
-	{ text: 'Recipes', header: true, type: 'learn', key: 'examples' },
-	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
-	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
-	{ text: 'Add backend services', slug: 'guides/backend', key: 'guides/backend' },
-	{ text: 'Add integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
-	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
-	{ text: 'More recipes', slug: 'recipes', key: 'guides/recipes' },
-
-	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
 	{
 		text: 'Astro Template Syntax',
 		slug: 'core-concepts/astro-syntax',
 		key: 'core-concepts/astro-syntax',
 	},
-	{
-		text: 'UI Frameworks',
-		slug: 'core-concepts/framework-components',
-		key: 'core-concepts/framework-components',
-	},
-	{ text: 'Routing', slug: 'core-concepts/routing', key: 'core-concepts/routing' },
-	{ text: 'Markdown & MDX', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
+
+	{ text: 'Built-ins', header: true, type: 'learn', key: 'examples' },
 	{
 		text: 'Content Collections',
 		slug: 'guides/content-collections',
 		key: 'guides/content-collections',
 	},
 	{
-		text: 'Scripts & Event Handling',
-		slug: 'guides/client-side-scripts',
-		key: 'guides/client-side-scripts',
+		text: 'View Transitions',
+		slug: 'guides/view-transitions',
+		key: 'guides/view-transitions',
+	},
+
+	{ text: 'Add-ons', header: true, type: 'learn', key: 'examples' },
+	{ text: 'Add integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
+	{
+		text: 'UI Frameworks',
+		slug: 'core-concepts/framework-components',
+		key: 'core-concepts/framework-components',
 	},
+	{ text: 'Server-side Rendering', slug: 'guides/server-side-rendering', key: 'guides/server-side-rendering' },
```

Comment:
This is my only complaint! Seems like Server-side Rendering should be under Built-ins. I know this section is getting revamped, but I don't think it's an add-on.

Reply:
Ah, I see there is discussion happening in Discord about this point.

Reply:
Mirroring discussion on Discord: because it requires adding an adapter, conceptually this falls closer to UI Frameworks because it doesn't work without something that needs to be `astro add`ed. So, literally, adding something for SSR to work **is** what makes it an add on. This category would be restricted to things very closely coupled to Astro's core offering, like our official packages where we also document their usage.

So noted, but as @delucis mentioned, from the outside looking in, "all you know is adding X makes SSR work" and I don't think this placement diminishes that it is something that is built into Astro core (but is helpful in understanding that it needs to be "unlocked" in order to actually use it.

And yes, I'd like this page to eventually be replaced with our Rendering Modes page, which maybe DOES go into basics, and then maybe we also keep an SSR page here that is devoted mostly to adding the adapter, marking individual pages in or out of pre-rendering etc.